src/bin/parser.js

```diff
@@ -205,18 +205,20 @@ export default class Harmonic {
         return this.templates[layout];
     }
 
-    async generatePosts(files) {
+    async generateFiles(files, filesType) {
```

Comment:
Maybe rename `filesType` -> `fileType`.
The parameter refers to a single file type that applies to all files, so both forms seem correct, though `fileType` somehow sounds better in my opinion.